x/observer/migrations/v11/migrate.go

```diff
@@ -0,0 +1,62 @@
+package v11
+
+import (
+	"slices"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/zeta-chain/node/pkg/chains"
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+type observerKeeper interface {
+	GetChainParamsList(ctx sdk.Context) (val types.ChainParamsList, found bool)
+	SetChainParamsList(ctx sdk.Context, chainParams types.ChainParamsList)
+}
+
+var skipBlockScanChainIDs = []int64{
+	// localnet
+	chains.GoerliLocalnet.ChainId,
```

Comment:
I suggest to keep it for the local evm because parsing all blocks will still be supported for at least one evm chains on each live network